Add support for box-decoration-break non-prefixed

kkirsche opened the pull request:
Add support for non-prefixed `box-decoration-break`
https://developer.mozilla.org/en-US/docs/Web/CSS/box-decoration-break

nschonni commented:
Thanks! Do you mind squashing the commits?
Also, could you add the vendor prefixed versions for `-webkit-` and `-o-`?

kkirsche commented:
@nschonni 

* `-webkit-box-decoration-break` already exists on [line 218](https://github.com/CSSLint/parser-lib/blob/master/src/css/Properties.js#L218).
* `-moz-box-decoration-break` exists on [line 209](https://github.com/CSSLint/parser-lib/blob/master/src/css/Properties.js#L209)
* `-o-box-decoration-break` is now added in this. 
* Standards `box-decoration-break` is been added in this. 

Commits have been rebased and squashed down to a single commit.

nschonni commented:
Thanks a bunch!
